manifests/certonly.pp

```diff
@@ -118,6 +118,17 @@
       ]
     }
 
+    'dns-ovh': {
+      require letsencrypt::plugin::dns_ovh
+      $_domains = join($domains, '\' -d \'')
+      $plugin_args = [
+        "--cert-name '${title}' -d",
+        "'${_domains}'",
+        "--dns-ovh-credentials ${letsencrypt::plugin::dns_ovh::config_dir}/dns-ovh.ini",
```

Comment:
Should the plugin declare a `$config_file` variable? Can be inside the body of the class. That way you don't rely on these two matching but can statically check it.

Reply:
I don't know exactly what you mean. Can you give me more details ?

Reply:
I meant something like (very shortened):
```puppet
class letsencrypt::plugin::dns_ovh {
  $config_file = "${letsencrypt::plugin::dns_ovh::config_dir}/dns-ovh.ini"

  file { $config_file:
    # ...
  }
}
```

Then you can use it here:
```puppet
"--dns-ovh-credentials ${letsencrypt::plugin::dns_ovh::config_file}",
```

Reply:
Ok, I changed $config_dir by $config_file in this PR